src/app/news/news-list-item.component.html

```diff
@@ -2,7 +2,7 @@
   <mat-card-header>
     <img mat-card-avatar [src]="item.avatar">
     <mat-card-title>
-      <a [routerLink]="['/users/profile', item.user.name]">
+      <a [routerLink]="['/users/profile', item.user.name, planetCode === item.messagePlanetCode ? {} : { planet: item.user.planetCode }]">
```

Comment:
This looks like a different issue entirely, similar to #5626 (updating that issue to include this).

Reply:
Also let's compare `planetCode` to `item.user.planetCode` for cases where the nation has put a message n the community team/enterprise/exchange feed.